src/uu/du/src/du.rs

```diff
@@ -175,7 +176,21 @@ fn du(
                             }
                         }
                     }
-                    Err(error) => show_error!("{}", error),
+                    Err(error) => match error.kind() {
+                        ErrorKind::PermissionDenied => {
```

Comment:
I don't have a better proposition for now (sorry!) but this is a lot of code just to show an error  :/

Reply:
Errors are hard though :(